src/index.js

```diff
@@ -55,6 +55,8 @@ export default function (CustomElement, opts) {
 
         if (name.indexOf('on') === 0) {
           syncEvent(node, name.substring(2), props[name]);
+        } else if (name === 'style') {
+          node.setAttribute('style', props[name]);
```

Comment:
@stevemao I added a test for `className` to ensure that works.

@bradleyayers it seems React actually doesn't do that. I was getting a false positive. Instead I decided to just pass-through style but we needed to make sure it was being passed as the `attrs` argument to `React.createElement()` for the test to pass.